airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -29,31 +29,42 @@
         "type": "string",
         "order": 2
       },
+      "schemas": {
+        "title": "Schemas",
+        "description": "The list of schemas to sync from. Defaults to user. Case sensitive.",
+        "type": "array",
+        "items": {
+          "type": "string"
+        },
+        "minItems": 1,
+        "uniqueItems": true,
+        "order": 3
```

Comment:
The most common Postgres schema is `public`, right? If that's the case, we can add a default value here.

```
"default": ["public"]
```

Reply:
done, thanks